Shelter.Java

```diff
@@ -0,0 +1,54 @@
+public class Shelter{
+/*The shelter class contains contact information for homeless shelters, 
+and some validation logic for verifying phone numbers
+*/
+    private String name;
+    private String location;
+    private int phoneNum;
+    private String desciption;
+    public Shelter(String name, String location, int phoneNum, String description){
```

Comment:
I would add an age requirement instance variable. This will make it easier to compare the eligibility a user has with a shelter based on their age.